src/Npgsql/NpgsqlBinaryImporter.cs

```diff
@@ -301,9 +301,11 @@ public void Complete()
                 _buf.EndCopyMode();
 
                 _connector.SendMessage(CopyDoneMessage.Instance);
-                Expect<CommandCompleteMessage>(_connector.ReadMessage());
+                var cmdComplete = Expect<CommandCompleteMessage>(_connector.ReadMessage());
                 Expect<ReadyForQueryMessage>(_connector.ReadMessage());
                 _state = ImporterState.Committed;
+                // Number of rows copied on success - see "Outputs" section of https://www.postgresql.org/docs/current/static/sql-copy.html
```

Comment:
This comment seems unnecessary, everything already seems very clear here.

Reply:
I agree, something from @chrisatx, I submitted this as a PR to @chrisatx's branch as well in case he wanted to continue.

Reply:
I agree with @roji, and we can remove this line.